docs/getting-started-guides/rkt/index.md

```diff
@@ -119,61 +122,101 @@ $ export KUBE_GCE_NODE_IMAGE=<image_id>
 $ export KUBE_CONTAINER_RUNTIME=rkt
 ```
 
-You can optionally choose the version of rkt used by setting `KUBE_RKT_VERSION`:
+Optionally, set the version of rkt by setting `KUBE_RKT_VERSION`:
 
 ```shell
 $ export KUBE_RKT_VERSION=1.9.1
 ```
 
-Then you can launch the cluster by:
+Optionally, select an alternative [stage1 isolator](#modular-isolation-with-interchangeable-stage1-images) for the container runtime by setting `KUBE_RKT_STAGE1_IMAGE`:
 
 ```shell
-$ cluster/kube-up.sh
+$ export KUBE_RKT_STAGE1_IMAGE=<stage1-name>
 ```
 
-### Launch a CoreOS/rkt cluster on AWS
+Then you can launch the cluster with:
 
-`kube-up` for AWS is currently unsupported.
-Instead, we recommend you to refer the [Kubernetes on AWS guide](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html) to launch a CoreOS/rkt cluster on AWS.
+```shell
+$ cluster/kube-up.sh
+```
 
-### Deploy apps to your cluster
+### Launch a rktnetes cluster on AWS
+
+The `kube-up` script is not yet supported on AWS. Instead, we recommend following the [Kubernetes on AWS guide](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html) to launch a CoreOS Kubernetes cluster on AWS, then setting kubelet options as above.
+
+### Deploy apps to the cluster
+
+After creating the cluster, you can start deploying applications. For an introductory example, [deploy a simple nginx web server](/docs/user-guide/simple-nginx). Note that this example did not have to be modified for use with a "rktnetes" cluster. More examples can be found in the [Kubernetes examples directory](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/).
+
+## Modular isolation with interchangeable stage1 images
+
+rkt executes containers in an interchangeable isolation environment. This facility is called the [*stage1* image](https://coreos.com/rkt/docs/latest/devel/architecture.html#stage-1). There are currently three supported rkt stage1 images:
+
+* `systemd-nspawn` stage1, the default. Isolates running containers with Linux kernel namespaces and cgroups in a manner similar to the default container runtime.
+* [`lkvm` stage1](https://coreos.com/rkt/docs/latest/running-lkvm-stage1.html), runs containers inside a KVM hypervisor-managed virtual machine.
```

Comment:
Additionally i would change lkvm stage1 to kvm stage1 cause we are working also on qemu integration (not only lkvm)